src/Lykke.Sdk/LykkeConfigurationOptions.cs

```diff
@@ -13,9 +13,16 @@ public class LykkeConfigurationOptions
         /// <summary>Default error handler.</summary>
         public CreateErrorResponse DefaultErrorHandler { get; set; }
 
+        /// <summary>API version</summary>
+        public string ApiVersion { get; set; }
+
+        /// <summary>Document title displayed in Swagger UI.</summary>
+        public string SwaggerDocumentTitle { get; set; }
```

Comment:
I think, will be better to localize swagger-specific options by moving this to separate options model and aggregate it here. Like it done in LykkeServiceOptions - https://github.com/LykkeCity/Lykke.Sdk/blob/master/src/Lykke.Sdk/LykkeServiceOptions.cs#L24